BraveWallet/Crypto/Search/AssetSearchView.swift

```diff
@@ -28,9 +28,7 @@ private struct TokenView: View {
 }
 
 struct AssetSearchView: View {
-  var tokenRegistry: BraveWalletERCTokenRegistry
-  var keyringStore: KeyringStore
-  var networkStore: NetworkStore
+  var walletStore: WalletStore
   
```

Comment:
why we pass the entire `walletStore` instead of just passing  a `@ObservedObject var tokenRegistery`?

Reply:
token registry isn't observable since there isn't a "store" wrapper around it